src/libmscore/segment.cpp

```diff
@@ -986,7 +986,7 @@ bool Segment::setProperty(Pid propertyId, const QVariant& v)
 qreal Segment::widthInStaff(int staffIdx, SegmentType t) const
 {
     const qreal segX = x();
-    qreal nextSegX = segX;
+    qreal nextSegX;
```

Comment:
Would you tell, why this change was made?

Reply:
In the lines here below, `nextSegX` will be set to another value anyway, so Xcode gave a warning that the initial value is never read. Therefore, I removed that initial value.

Reply:
Please, return initial values to all variables with primitive types. Variables with undefined values are very dangerous. After some time implementation of this function may change and this initial value can save from hard reproduced bug

Reply:
I see, I returned it!